Docs pm 2.9 dh (#143)
mulesoft/docs-partner-manager: modules/ROOT/pages/inbound-message-flows.adoc

== Inbound Message Flows

> 

Reviewer: Should have edited this first....

> An inbound message flow receives application messages from your partner, either directly or through a 3rd party connection, translates the messages to the format used by your backround applications, and forwards the messages to those applications.

Reviewer: GLOBALLY search for and replace "3rd party connection" with "third-party connection."

Reviewer: Because one list item has an internal comma, use semicolons to separate list items:
```suggestion
An inbound message flow receives application messages from your partner, either directly or through a third-party connection; translates the messages to the format used by your backround applications; and forwards the messages to those applications.
```

Reviewer: Where does "backend" come into this ^? That is the term used subsequently.

The following diagram shows the components of an inbound message flow:

image::pm-inbound-message-flow.png[Inbound message flow]

|===
|Setting |Description |Supported Formats or Protocols

> |Receiving endpoint | Endpoint where the host receives transmissions from the trading partner a|

Reviewer: parallelism with target endpoint desc:
```suggestion
|Receiving endpoint | Endpoint at which a host receives transmissions from a trading partner a|
```

* AS2
* FTP
* SFTP
* HTTP
* HTTPS

> |Source message type |Message type of the input payload a|

Reviewer: Is "input payload" = "received message payload"?
It would be best to be really, really careful with terms in PM doc, because they are complicated, at least for me....

Author: Good suggestion

* X12
* EDIFACT
* JSON
* XML

|Transform |DataWeave map used to transform the source messsage type to the target message type a| N/A

> |Target message type |Format into which the DataWeave map transforms received messages a|

Reviewer: Or, for parallelism with Line 19:
```suggestion
|Target message type | Message type into which DataWeave map transforms the source message a|
```

* JSON
* XML

> |Target endpoint | Endpoint to which the translated message is sent (typically, a process API)

Reviewer: ```suggestion
|Target endpoint | Endpoint to which the transformed received message is sent (typically, a process API)
```
Is this the backend?

Reviewer: I would go on to use these precise terminologies/definitions throughout to avoid confusion among message types (source, target) and endpoints (receiving, target).

Reviewer: Note also that Create an Incoming... names three endpoints: _source_ at host, _target_ at host, and _receive_ from partners. Very confusing.
